### DISCUSSION TITLE
Just a quick question about the Fair Use policy on the Free Tier

### DISCUSSION THREAD

**Discussion by maddsua** [vercel/community#2208] [Help]
### Summary

Let's say I create a few projects on the free tier, just to showcase my skills. On their own, they are not commercial by no mean. So the questions is:

1. Can I link to them from my website (not on Vercel) to, well, showcase?
2. If a project looks like a real prod app, but in reality is just a demo, is there a chance that a Vercel employee could mistakenly nuke it ?

I find the Fair Use policy kinda confusing. On the one hand, the projects cannot be considered as commercial products, but they are the way I "hunt" for clients, therefore they make me money.

The core issue is that we aren't made of money, and paying 20 bucks/month for 50 visitors and a total of 250mb bandwidth used (of 1Tb provided) does not seem like my wisest decision.

**Comment by leandro-n-ortiz** [comment]
That's a good question, because there is a topic saying it could be used to host portfolio: https://github.com/orgs/vercel/discussions/1376
But as we can make money with it, I'm not sure whether the fair use policy really allows that...
